Snakes.py

```diff
@@ -157,10 +190,13 @@ def gameLoop():
 
         if foodX==x and foodY==y:
             foodX=round(random.randrange(0,790)/10.0)*10.0
-            foodY=round(random.randrange(0,590)/10.0)*10.0
+            foodY=round(random.randrange(20,590)/10.0)*10.0
             snakeLen+=1
-            sound=pygame.mixer.Sound("Assets\Sounds\Point.wav")
+            blocks(foodX,foodY,snakeLen-1)
+            sound=pygame.mixer.Sound("Point.wav")
```

Comment:
Please rebase and then change this path to the f]proper file present in assets folder